docs/core/install/linux-package-manager-centos7.md

```diff
@@ -55,3 +55,11 @@ sudo yum install dotnet-runtime-3.1
 ## How to install other versions
 
 [!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
+
+## Troubleshoot the package manager
+
+This section provides information on common errors you may get while using the package manager to install .NET Core.
+
+### Failed to fetch
+
+While installing the .NET Core package, you may see an error similar to `Failed to fetch ... File has unexpected size ... Mirror sync in progress?`. Generally speaking, this error means that the package feed for .NET Core is being upgraded with newer package versions, and that you should try again later. During an upgrade, the package feed should not be unavailable for more than 2 hours. If you continually receive this error for more than 2 hours, please file an issue at <https://github.com/dotnet/core/issues>.
```

Comment:
The error's actually different for RPM repository clients, sorry I missed adding this to the actual doc issue: https://github.com/dotnet/core/issues/4167.

I think this would work:

> similar to `Signature verification failed for file 'repomd.xml' from repository 'packages-microsoft-com-prod'`.